index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio project</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <header>
+        <div class="header-container">
+            <h1>My Logo</h1> 
+            <img src="./images/menu.png" width="40" height="40">
+        </div>
```

Comment:
- [ ] Kindly wrap your `hamburger menu` in a `nav` tag 🙏 . The `nav` tag is for semantic reasons.